stats/opentelemetry/client_metrics.go

```diff
@@ -174,7 +208,23 @@ func (csh *clientStatsHandler) processRPCEnd(ctx context.Context, mi *metricsInf
 		st = canonicalString(s.Code())
 	}
 
-	clientAttributeOption := metric.WithAttributes(attribute.String("grpc.method", ci.method), attribute.String("grpc.target", ci.target), attribute.String("grpc.status", st))
+	attributes := []attribute.KeyValue{
+		attribute.String("grpc.method", ci.method),
```

Comment:
Should `metricsInfo` be named `attemptInfo` to distinguish it better from `callInfo`?  It took me awhile to figure this out while reviewing this change.

Reply:
It's scoped to call server side, and it uses the same. I'd like to change it, but do you have a better name that works for both sides?

Reply:
Attempts and calls are the same thing server-side.

Do we use both of these on the server or just the one?

Reply:
We only use this on the server side, CallInfo is only used client side. If call info is the same as AttemptInfo then I can Switch this name to AttemptInfo.

Reply:
Ohhh, I'm sorry, I thought this was storing per-attempt data on the client side, vs. per-call data.  So it took me awhile to figure out and I got it wrong...

Maybe they should be `clientCallInfo` and `serverCallInfo` then?

Reply:
This is storing per attempt info. The Client Call Info has a specific type: https://github.com/grpc/grpc-go/pull/7256/files#diff-6cb099cd7cc7853ea6b069e48ce1493ae735d7a56f7e36a8a828b20d82470eddR176